applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -354,12 +366,40 @@ config BT_AUDIO_BROADCAST_ID_FIXED
 	  Fixed broadcast ID; 3 octets. Will only be used if BT_AUDIO_USE_BROADCAST_ID_RANDOM=n.
 	  Should only be used for debugging.
 
+config BT_AUDIO_BROADCAST_IMMEDIATE_FLAG
+	bool "Immediate rendering flag"
+	depends on TRANSPORT_BIS
+	default n
+	help
+	  Set the immediate rendering flag.
+
 config AURACAST
 	bool "Enable Auracast"
 	default y
 	help
 	  When Auracast is enabled a Public Broadcast Announcement will be included when advertising.
 
+menu "QoS"
+
+config BT_AUDIO_PRESENTATION_DELAY_US
+	int "Presentation delay"
+	default 10000
+	help
+	  Presentation delay for the ISO link.
+
+config BT_AUDIO_MAX_TRANSPORT_LATENCY_MS
```

Comment:
We could add a range in here:
For max transport latency in here, it is from 0x0005 to 0x0FA0.
Ref: Core specification 5.3, p2575